src/main/assembly/assembly.xml

```diff
@@ -33,6 +33,7 @@
       <excludes>
         <exclude>ocp/</exclude>
         <exclude>ds-image-local.yaml</exclude>
+        <exclude>ds-image-kaltura-integration-test.yaml</exclude> <!-- Not sute why this test file is in the main configuration folder. -->
```

Comment:
That's an error. Could you move it to test resources?

Reply:
Sorry, but you seem to have deleted the file so now `KalturaApiIntegrationTest` won't run. I was hoping you would move it to test resources and adjust the path in the `callKalturaApi` unit test method accordingly.